setup.cfg

```diff
@@ -1,2 +1,2 @@
 [flake8]
-ignore = E741,E226,W503,W504
+ignore = E741,E226,W503,W504,E402
```

Comment:
Allows imports that don't happen at the beginning of the file. Useful for the `__init__.py` files.

Reply:
Humm, I'm worried about this. I'd rather add noqas where we know that it's fine to break this rule. Otherwise flake will be silent when we're doing silly things.

Reply:
Not importing a module at the top of a file is not a big deal, it's just stylistic.
E402 is primarily included there just so I can make all `__init__.py` files flake-compatible.

For example, in `pyccl/__init__.py` at the top of the file we have:
```python
try:
    __version__ = get_distribution(__name__).version
except DistributionNotFound:
    # package is not installed
    pass
del get_distribution, DistributionNotFound
```
which causes E402 to be raised everywhere throughout the file. Not a good idea to add noqas in every line there.

Just to be clear, this is the complain you get from flake:
`E402 module level import not at top of file`

Reply:
But you're not making the files flake compatible, right? You're just forcing flake to ignore E402.
I'd rather explicitly add `# flake8: noqa E402` to the top of the `__init__` files than have flake ignore this everywhere (since it may be useful for it to point it out when we add more code)

Reply:
done